docs/apis/subsystems/external/writing-a-service.md

```diff
@@ -316,10 +316,10 @@ Because some web service protocols are strict about the number and types of argu
 ```php
 public static function get_biscuit_parameters() {
     return new external_function_parameters([
-        'chocolatechips' => new external_value(PARAM_BOOL, PARAM_REQUIRED),
-        'glutenfree' => new external_value(PARAM_BOOL, PARAM_DEFAULT, false),
+        'chocolatechips' => new external_value(PARAM_BOOL, 'if biscuit contains chocolate chips', PARAM_REQUIRED),
```

Comment:
If you don't mind, can you look at reformatting these as follows:

```suggestion
        'chocolatechips' => new external_value(
            PARAM_BOOL,
            'if biscuit contains chocolate chips',
            PARAM_REQUIRED,
        ),
```

The reason for this is that when it's a long line, it wraps and becomes less readable on Mobile.

Where we can, I try to wrap code samples at an earlier point for docs to make it more readable for those narrower screen situations.

Reply:
sure, no problem